### PR TITLE
added mozilla addon link to the readme

### PR DESCRIPTION
**Pull request by ArjixWasTaken**
and improved the readme a bit, just a tiny bit

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
image

**Comment by ArjixWasTaken** [comment]
unless the mozilla addon is unofficial...i dont really know

**Comment by ArjixWasTaken** [comment]
PS: the mozilla image I used is an edit of chrome's badge, and I allow you to use it

**Comment by QasimWani** [comment]
This is cool. Yeah the Firefox version is a fork developed by @njelich which is community driven. There's no such thing as official and official. Pretty decentralized when it comes to the that 😉

File: README.md [hunk 1]
```diff
@@ -1,40 +1,44 @@
 <h1 align="center">
   <a href="https://standardjs.com"><img src="assets/octocode.png" alt="LeetHub - Automatically sync your code to GitHub." width="400"></a>
   <br>
-  <a href="https://chrome.google.com/webstore/detail/leethub/aciombdipochlnkbpcbgdpjffcfdbggi">LeetHub</a> - Automatically sync your code to GitHub.
+  <a href="https://github.com/QasimWani/LeetHub">LeetHub</a> - Automatically sync your code to GitHub.
```
Comment (QasimWani) [inline README.md:4]: replace with chrome webstore URL – https://chrome.google.com/webstore/detail/leethub/aciombdipochlnkbpcbgdpjffcfdbggi

**Review by QasimWani — CHANGES_REQUESTED** [review]

**Comment by aashu** [comment]
@QasimWani I tried loading leethub directly from this repo and it was working without any issues. So I don't think there is any requirement to maintain a seperate fork for firefox. Also, the fork you are talking about doesn't seems to be updated since April. It would be great if we can use this repo only and list extension to firefox addons store.

**Comment by njelich** [comment]
@aashu Yep, my changes added compatibility and it was merged to the main repo.

**Pull request merged**